### PR TITLE
Updated credits. Added copyright notice. PluginAPI to add plugin credits.

### PR DESCRIPTION
**Pull request by alexhliu**
image
image
image

### CODE REVIEW

File: chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java [hunk 1]
```diff
@@ -40,10 +39,13 @@
  */
 public class ChunkyFx extends Application {
 
+  private static HostServices hostServices;
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java:42]: They are used at one place, we don't really need a field then

File: chunky/src/java/se/llbit/chunky/ui/CreditsController.java [hunk 2]
```diff
@@ -43,4 +129,26 @@ public void setStage(Stage stage) {
       }
     });
   }
+
+  private void launchAndReset(Hyperlink link, String url) {
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/ui/CreditsController.java:133]:
```suggestion
  private void openLink(Hyperlink link, String url) {
```

File: chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java [hunk 3]
```diff
@@ -75,4 +77,12 @@ public static void startChunkyUI(Chunky chunkyInstance) {
     ChunkyFx.chunkyInstance = chunkyInstance;
     launch();
   }
+
+  /**
+   * Launch a url in the default browser.
+   */
+  @PluginApi
+  public static void launchUrl(String url) {
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java:85]:
```suggestion
  public static void openUrl(String url) {
```

File: chunky/src/res/se/llbit/chunky/ui/Chunky.fxml [hunk 4]
```diff
@@ -151,7 +152,16 @@
           <Hyperlink fx:id="discordLink" text="Discord Server" />
           <Hyperlink fx:id="guideLink" text="jackjt8's Guide to Chunky" />
           <Separator />
-          <Label text="Chunky was created by Jesper Öqvist (jesper@llbit.se)"/>
+          <Label wrapText="true" text="Chunky was created by Jesper Öqvist (jesper@llbit.se)"/>
+          <Label wrapText="true" text="Chunky is Copyright (c) 2010-2021, Jesper Öqvist and Chunky contributors." />
+          <FlowPane>
+            <Label wrapText="true" text="Permission to modify and redistribute is granted under the terms of the " />
+            <Hyperlink fx:id="gplv3" wrapText="true" text="GPLv3 license.">
```
Comment (leMaik) [inline chunky/src/res/se/llbit/chunky/ui/Chunky.fxml:159]: We could write the full license name here. Nit: Move the `.` out of the link

File: chunky/src/res/se/llbit/chunky/ui/Credits.fxml [hunk 5]
```diff
@@ -8,81 +8,141 @@
...
+                              <children>
+                                 <Label text="a_Tom" />
+                                 <Label text="TOGoS" />
+                                 <Label text="Redox (alexhliu)" />
+                                 <Label text="jackjt8" />
+                                 <Label text="073TheBoss" />
+                                 <Label text="NotStirred" />
+                                 <Label text="Ice Selkie (StanleyMines)" />
+
+                                 <Hyperlink fx:id="ghContributors" text="Full List" />
```
Comment (leMaik) [inline chunky/src/res/se/llbit/chunky/ui/Credits.fxml:69]:
```suggestion
                                 <Hyperlink fx:id="ghContributors" text="Show more…" />
```

File: chunky/src/res/se/llbit/chunky/ui/Credits.fxml [hunk 6]
```diff
@@ -8,81 +8,141 @@
...
+                        </padding>
+                        <children>
+                           <Label text="Tools and Libraries">
+                              <font>
+                                 <Font name="System Bold" size="12.0" />
+                              </font>
+                           </Label>
+                           <VBox>
+                              <children>
+                                 <Hyperlink fx:id="markdown" text="Markdown">
```
Comment (leMaik) [inline chunky/src/res/se/llbit/chunky/ui/Credits.fxml:97]: Where does chunky use that btw.? Would be cool to have markdown for the changelogs in the launcher!

**Review by leMaik — CHANGES_REQUESTED** [review]
Looks great, also thanks for the promotion :rofl:

File: chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java (hunk 3 above)
Comment (alexhliu) [reply chunky/src/java/se/llbit/chunky/ui/ChunkyFx.java]: 👍

File: chunky/src/res/se/llbit/chunky/ui/Chunky.fxml (hunk 4 above)
Comment (alexhliu) [reply chunky/src/res/se/llbit/chunky/ui/Chunky.fxml]: 👍

File: chunky/src/res/se/llbit/chunky/ui/Credits.fxml (hunk 5 above)
Comment (alexhliu) [reply chunky/src/res/se/llbit/chunky/ui/Credits.fxml]: 👍

**Pull request merged**